docs/source/author/topics.rst

```diff
@@ -80,1007 +76,634 @@ the project's topic application, where:
+This section details configuration files within the ``content`` directory for a specific
+language.
+These files are in YAML format. If you are not familiar with YAML, see
+:doc:`understanding_configuration_files`.
 
-      programming-exercises: programming-exercises/programming-exercises.yaml
+The diagram below shows an example of YAML file locations for the
+``content/en/`` language directory, where:
 
-- ``follow-up-activities`` - A configuaration file defining follow up
-  activities for the topic.
-  The file path is relative to the YAML file.
+- Blue is directories.
+- Red is YAML configuration files.
 
-  - For example:
+.. raw:: html
+  :file: ../_static/html_snippets/topics_content_directory_tree_only_yaml.html
 
-    .. code-block:: yaml
+The diagram below shows an overview of what is in each config file:
 
-      follow-up-activities: follow-up-activities/follow-up-activities.yaml
+.. image:: ../_static/img/topics_config_file_overview_diagram.png
 
-- ``other-resources-md-file`` - A Markdown file describing other resources.
-  The file path is relative to the YAML file.
+In the following sections, each configuration file is exaplained in more detail.
 
-  - For example:
+.. note::
+  
+  - We use the term "key" to specify a field name. Keys map to particular values (which
+    range from learning outcome text, to the structure and attributes of a lesson).
+    There are two things to remember with keys:
 
-    .. code-block:: yaml
+    - They must be written in lowercase and use hyphens instead of spaces.
 
-      other-resources-md-file: other-resources.md
+    - They must be exact matches to work, for example, if you name a lesson
+      ``bits-and-bytes``, referencing it in another configuration file as
+      ``bytes-and-bits`` will raise an error.
 
-A complete topic file may look like the following:
+.. note::
 
-.. code-block:: yaml
+  - Some of the keys have angle brackets around them, ``<like so>``. This means that they
+    are variables and you can call them whatever you like in your configuration file
+    (without the angle brackets).
 
-  slug: binary-numbers
-  md-file: index.md
-  icon: img/binary-numbers-0-1.png
-  unit-plans:
-    - unit-plan/unit-plan.yaml
-  programming-exercises: programming-exercises/programming-exercises.yaml
-  follow-up-activities: follow-up-activities/follow-up-activities.yaml
-  other-resources-md-file: other-resources.md
 
-.. _unit-plan-file:
+.. _application-structure-file:
 
-Unit Plan File
+Application Structure Configuration File
 ------------------------------------------------------------------------------
 
-This file defines the attributes of a unit plan, including all lessons (and
-their respective attributes) for the unit plan.
-
-The location of this file is within a unit plan directory.
-These files are listed in a :ref:`topic-file`.
+- **File Name:** ``structure.yaml``
 
-This file must contain the following key/value pairs:
+- **Location:** ``topics/content/<language>/``
```

Comment:
Same angle bracket syntax as mentioned above? Would it be worth using a different bracket type?